build.sh

```diff
@@ -44,9 +44,11 @@ HTML="<html>
     </style>
   </head>
   <body>
-  This is a list of custom Scalars specifications, contributed by the community.
+    <p>This is a directory of GraphQL Custom Scalar specifications, contributed by the community.</p>
+
+    <p>Specifications in this directory can be referred to with the <a href=\"https://spec.graphql.org/draft/#sec--specifiedBy\">@specifiedBy directive</a>, a built-in directive for documenting the behavior of custom scalar types.</p>
```

Comment:
Adding a note this is intended to be used in conjunction with the `@specifiedBy` directive